graviti/dataframe/frame.py

```diff
@@ -573,3 +580,10 @@ def extend(self, objs: Union[Iterable[Dict[str, Any]], "DataFrame"]) -> None:
         2   d.jpg    4      4
 
         """
+        if not isinstance(objs, self.__class__):
+            objs = self.__class__.from_pyarrow(pa.array(objs, self.schema.to_pyarrow()))
+        else:
+            if self.schema != objs.schema:
```

Comment:
Portex schema do not have `__eq__` method.
Also here checking the pyarrow type is enough